README.md

```diff
@@ -0,0 +1,47 @@
+# Development setup
+Please make sure you have a proper *python* setup with version >= 3.12.
+Another prerequisite is *poetry* since we use it for dependency management.
+If it's not already installed on your system, have a look at https://python-poetry.org/docs/.
+
+## Install dependencies
+```bash
+poetry install --with dev
+```
+
+## Database
+This project expects the data of [nmshd-backbone](https://github.com/nmshd/backbone) to be available in a MSSQL database.
+For easier development setups, the *docker-compose.yml* avaiable in *docker/development* provides a way to automatically load *.bacpac* files and provides a MSSQL server containing this backup.
+For now, the database used within the development environment uses the default mssql port (1433) and the default mssql user (sa).
+The password for this user, the database name and the path to the backup file have to be specified in the environment before starting the dev stack:
+```bash
+MSSQL_PASSWORD=Is1This2The3Real4Life?  # Password for the sa database user
+MSSQL_DB=MyDatabase  # Name of the database to which backup is imported
+DB_BACKUP_FILE=MyDatabaseBackup.bacpac  # path to database backup
```

Comment:
Sollte noch ein `export ` hin, sonst kopieren das Leute in ihre Shell und nichts passiert :(